scripts/exrrfs_mpassit.sh

```diff
@@ -34,6 +34,22 @@ YYYYMMDDHH=${CDATE}
 YYYYMMDD=${CDATE:0:8}
 HH=${CDATE:8:2}
 #
+# find init.nc
+#
+n=0
+while [[ $n -le 12 ]]; do
+  CDATEic=$($NDATE -$((10#${n})) ${CDATE})
+  YYYYMMDDic=${CDATEic:0:8}
```

Comment:
`YYYYMMDD` is called `PDY` based on the NCO standard.
You may change `YYYYMMDDic` to `PDYic`